srcCpp/CustomType.cxx

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
@alfonso-soto and @softwarejimenez, is it a good practice to protect the whole file by adding:
```
#ifdef RTI_CUSTOM_TYPE
```

I don't think it is needed, but I would add it for this file and the .h, so even any IDE would show these functions as not available.

Thoughts?

Reply:
I like the idea. 
Done